migrations/auth/1588740228_add_anonymous_identity.down.sql

```diff
@@ -0,0 +1 @@
+-- Put downgrade SQL here
```

Comment:
Should we have downgrade migration?

Reply:
Sorry forgot about it, will fix it later.